server/api.go

```diff
@@ -95,13 +101,34 @@ func (p *Plugin) connectUserToGitHub(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	conf := p.getOAuthConfig()
+	privateAllowed := false
+	pVal := r.URL.Query().Get("private")
+	if pVal == "ok" {
```

Comment:
Would it make sense to rely on [`strconv.ParseBool`](https://golang.org/pkg/strconv/#ParseBool) for something like this instead of a string value of `ok` for a boolean query string parameter?